app/controllers/bookings_controller.rb

```diff
@@ -1,46 +1,52 @@
 class BookingsController < ApplicationController
-  before_action :find_dress, only: %i[new show create destroy]
+  before_action :find_dress, only: %i[new create destroy]
 
```

Comment:
Would it make sense to have a `before_action :find_booking` for your bookings controller? That might help keep things a bit more DRY. 

```rails
def find_booking 
  @booking = Booking.find(params[:id])
end
```

Reply:
Good idea - for now I will just merge it because things are getting a bit long and a lot going on so if updated can you just push it on a new branch :)

Reply:
Thanks Dillion, I'll commit that change